include/SDL3/SDL_gpu.h

```diff
@@ -1790,13 +1783,13 @@ extern SDL_DECLSPEC void SDLCALL SDL_SetGPUTextureName(
  *
  * Useful for debugging.
  *
- * \param commandBuffer a command buffer.
+ * \param command_buffer a command buffer.
  * \param text a UTF-8 string constant to insert as the label.
  *
  * \since This function is available since SDL 3.0.0.
  */
 extern SDL_DECLSPEC void SDLCALL SDL_InsertGPUDebugLabel(
-    SDL_GPUCommandBuffer *commandBuffer,
+    SDL_GPUCommandBuffer *command_buffer,
```

Comment:
buffer?

Reply:
I think it's important to disambiguate the term "buffer", it's an extremely overloaded term especially in graphics.

Reply:
I do understand that the type disambiguates it but I think it's still confusing. For example there are functions in this API that use a command buffer, a transfer buffer, and a buffer all at the same time.